.github/workflows/docs.yml

```diff
@@ -0,0 +1,55 @@
+name: docs
+
+defaults:
+  run:
+    shell: bash
+    working-directory: docs
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'docs/**'
+      - '.github/workflows/docs.yml'
```

Comment:
Is this not implicit? (I am not sure either way, but would like to verify so as not to spread this pattern if it is unnecessary.)

Reply:
It is required.  My first commits only included the docs path and thus _did not_ trigger the workflow until I added this line.